includes/abstracts/class-evf-form-fields.php

```diff
@@ -219,10 +222,15 @@ public function field_element( $option, $field, $args = array(), $echo = true )
 			case 'row':
 				$output = sprintf( '<div class="everest-forms-field-option-row everest-forms-field-option-row-%s %s" id="everest-forms-field-option-row-%s-%s" data-field-id="%s">%s</div>', $slug, $class, $id, $slug, $id, $args['content'] );
 				break;
-
+			// Icon.
```

Comment:
Please restore these spaces, they adhere to WPCS

Reply:
Okay, sorry about that.